src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -193,12 +193,19 @@ function duplicateLayer() {
                 v-if="hasDataDisclaimer"
                 :complete-disclaimer-on-click="true"
                 :source-name="attributionName"
+                :is-external-data-local="hasDataDisclaimerLocal"
             >
                 <FontAwesomeIcon
-                    ref="tooltipAnchor"
-                    class="data-disclaimer-tooltip text-primary p-2"
-                    icon="user"
-                    data-cy="menu-external-disclaimer-icon"
+                    v-if="hasDataDisclaimer && !hasDataDisclaimerLocal"
```

Comment:
no need to test for `hasDataDisclaimer` as the parent is only added when it's `true`

you could then test for the validity of `hasDataDisclaimerLocal` and add a `v-else` to the other icon